patient_case_summary/patient_case_summary.py

```
@@ -0,0 +1,788 @@
+     - Consider partial matches. For "Atopic dermatitis (disorder)", a medication used for allergic conditions or skin inflammations could be relevant.
+3. Ignore patient demographics for relevance determination; they are just context.
+4. Return the final output strictly as a JSON object following the schema (provided as a tool call). 
+   Do not include extra commentary outside the JSON.
+
+**Patient Data**:
+{patient_info}
+"""
+
+async def create_condition_bundles(
+    patient_data: PatientInfo, llm: Optional[LLM] = None
+) -> ConditionEncounters:
```

Ruff (0.8.2)

Undefined name `ConditionEncounters`

(F821)